client/config/crd/groupsnapshot.storage.k8s.io_volumegroupsnapshotcontents.yaml

```diff
@@ -17,6 +17,15 @@ spec:
     - vgscs
     singular: volumegroupsnapshotcontent
   scope: Cluster
+  conversion:
+    strategy: Webhook
+    webhook:
+      conversionReviewVersions: ["v1","v1beta1"]
+      clientConfig:
+        service:
+          namespace: default
+          name: snapshot-webhook-service
+          path: /convert
```

Comment:
If someone deploys the CRD as it is, it will require the conversion webhook to run.
That complicates my local tests - I just want v1beta2 and I want to run it quickly, without generating TLS keys and whatnot. Would it be possible to make the conversion opt-in (or opt-out) via kustomize? And is it worth the effort?